tests/cli.rs

```diff
@@ -168,8 +170,10 @@ async fn compare_results(expected_path: &Path, actual: &str) -> Result<()> {
         .with_context(|| format!("failed to read result file {expected_path:?}"))?;
 
     if normalize_string(&expected) != normalize_string(actual) {
+        let expected = normalize_string(&expected);
```

Comment:
Should these get set, then checked with the `if`, and printed if necessary? Or is the normalization lightweight enough to do twice?